src/cmd/vm.rs

```diff
@@ -241,15 +241,16 @@ fn run_start(args: &ArgsStart) -> Result<()> {
 }
 
 fn find_betty_script(arc: &Option<String>) -> Result<String> {
-    let path = arc
+    let arc_path = arc
         .clone()
         .unwrap_or_else(|| env::current_dir().unwrap().to_string_lossy().to_string());
+    let betty_path = format!("{}/tools/vendor/google_prebuilts/arc", arc_path);
```

Comment:
Maybe using Path::join instead of format! is better.
https://doc.rust-lang.org/std/path/struct.Path.html#method.join